src/main/java/org/openhab/binding/zwave/handler/ZWaveSerialHandler.java

```diff
@@ -369,7 +371,7 @@ public void sendPacket(SerialMessage serialMessage) {
                 logger.debug("Message SENT");
             }
         } catch (IOException e) {
-            logger.error("Got I/O exception {} during sending. exiting thread.", e.getLocalizedMessage());
+            logger.debug("Got I/O exception {} during sending. exiting thread.", e.getLocalizedMessage());
```

Comment:
Probably this should be WARN - this is quite a serious error.

Reply:
Changed to warn.